### PR TITLE
[6.5.x] DROOLS-2777: Guided Decision Table is changing date field value based on the timezone

### PR DESCRIPTION
**Pull request by karreiro**
See: https://issues.jboss.org/browse/DROOLS-2777

---

_"Cherry picking"_ https://github.com/kiegroup/kie-wb-common/pull/2038 on `6.5.x` branch. We don't need a PR for `drools-wb` in the `6.5.x`.

### CODE REVIEW

**Comment by karreiro** [comment]
Jenkins, please retest this.

**Comment by karreiro** [comment]
Jenkins please retest this

File: kie-wb-common-widgets/kie-wb-decorated-grid-widget/src/test/java/org/kie/workbench/common/widgets/decoratedgrid/client/widget/cells/PopupDateEditCellTest.java [hunk 1]
```diff
@@ -54,8 +84,14 @@
 
     @Before
     public void setup() {
-        cell = new PopupDateEditCell(DateTimeFormat.getFormat(DateTimeFormat.PredefinedFormat.DATE_LONG),
-                                     false) {
+        System.setProperty("user.timezone", "Europe/Vilnius");
```
Comment (jomarko) [inline kie-wb-common-widgets/kie-wb-decorated-grid-widget/src/test/java/org/kie/workbench/common/widgets/decoratedgrid/client/widget/cells/PopupDateEditCellTest.java]: Same comments about system properties cleanup as in #2038

**Comment by karreiro** [comment]
Jenkins please retest this

**Comment by karreiro** [comment]
Jenkins please retest this

**Comment by karreiro** [comment]
@jomarko Changes applied.

**Pull request merged**